tails_files/install.sh

```diff
@@ -1,17 +1,37 @@
 #!/bin/bash
+# SecureDrop persistent setup script for Tails 
 
 # check for root
 if [[ $EUID -ne 0 ]]; then
-    echo "This script must be run as root" 1>&2
-    exit 1
+	echo "This script must be run as root" 1>&2
+	exit 1
 fi
 
-PERSISTENT=/home/amnesia/Persistent
+# set paths and variables
+HOMEDIR=/home/amnesia
+PERSISTENT=$HOMEDIR/Persistent
 INSTALL_DIR=$PERSISTENT/.securedrop
 ADDITIONS=$INSTALL_DIR/torrc_additions
 SCRIPT_PY=$INSTALL_DIR/securedrop_init.py
 SCRIPT_BIN=$INSTALL_DIR/securedrop_init
 DOTFILES=/live/persistence/TailsData_unlocked/dotfiles
+TAILSCFG=/live/persistence/TailsData_unlocked
+DESKTOP=$HOMEDIR/Desktop
+ANSIBLE=$PERSISTENT/securedrop/install_files/ansible-base
```

Comment:
@ageis Looks like this is handled with the check `if [ ! -d "$ANSIBLE" ]` on line 30.